sys/src/9/amd64/archamd64.c

```diff
@@ -192,13 +192,16 @@ cpuidhz(uint32_t *info0, uint32_t *info1, CpuHypervisor hypervisor)
 //print("msr 2a is 0x%x >> 22 0x%x\n", rdmsr(0x2a), rdmsr(0x2a)>>22);
 			break;
 		case 0x000306a0:		/* i7,5,3 3xxx */
+		case 0x000306f0:		/* i7,5,3 5xxx and Xeon dev machine */
```

Comment:
Looking at this, I think a rebase might be a good idea. This change was done earlier?

Reply:
The case statement moved from a place where it would try MSR 0xCD to here, so that it won't try that (which cased a GPF on my dev machine).